### PR TITLE
Include modules manifest

### PR DESCRIPTION
**Pull request by Fryguy**
Follow up to #1671.  This adds the modules manifest list in addition to the packages list similar to ManageIQ/manageiq-ui-classic#7319

### CODE REVIEW

File: config/webpack.prod.js [hunk 1]
```diff
@@ -9,6 +9,52 @@ const sortBy = require('lodash/sortBy');
...
+
+  var modules = transformModules(data.modules);
+  modules = [...new Set(modules)]; // uniq
+
+  return modules;
+}
+
+const packagesFromModules = (modules) => {
+  // find the package.json file for each module
+  modules = modules.filter(m => m.includes("node_modules"));
```
Comment (himdel) [inline config/webpack.prod.js:25]: modules is a global here, missing `const`?

File: config/webpack.prod.js [hunk 2]
```diff
@@ -31,50 +77,25 @@ config.plugins.push(
...
+  }),
+
+  // Write out dependent packages list for audit purposes
+  new StatsWriterPlugin({
+    filename: "webpack_packages_manifest.json",
+    fields: ["modules"],
 
+    transform(data) {
+      modules = uniqueWebpackModules(data);
+      packages = packagesFromModules(modules);
```
Comment (himdel) [inline config/webpack.prod.js:98]: and here, both of them

File: config/webpack.prod.js (hunk 1 above)
Comment (Fryguy) [reply config/webpack.prod.js:25]: ugh I hate javascript 😆

Comment (Fryguy) [reply config/webpack.prod.js:25]: @himdel Oh wait, is it?  I thought it's just updating the incoming parameter.  I'm not introducing a new variable.

**Comment by Fryguy** [comment]
@himdel I updated the one comment, but I'm unsure on the other one.

File: config/webpack.prod.js (hunk 1 above)
Comment (himdel) [reply config/webpack.prod.js:25]: oh yeah, you're right `modules` is a param here

Comment (himdel) [reply config/webpack.prod.js:25]: travis went green so all is good now :)

**Comment by himdel** [comment]
LGTM :+1: :)

`manageiq/public/ui/service/webpack_packages_manifest.json`:

```json
[
  {
    "name": "@manageiq/font-fabulous",
    "license": "Apache-2.0",
    "version": "1.0.3",
    "location": "./node_modules/@manageiq/font-fabulous/package.json"
  },
...
```

`manageiq/public/ui/service/webpack_modules_manifest.json`:

```json
[
  "./node_modules/moment/moment.js",
  "./node_modules/d3-geo/src/math.js",
  "(webpack)/buildin/global.js",
  "./client/assets/sass/_explorer.sass",
...
```

**Comment by simaishi** [comment]
Jansa backport details:

```
$ git log -1
commit 1f5abddf58d6ddba27d66f5933bb21b86fd00f62
Author: Martin Hradil <mhradil@redhat.com>
Date:   Fri Sep 18 21:35:43 2020 +0000

    Merge pull request #1673 from Fryguy/include_modules_manifest

    Include modules manifest

    (cherry picked from commit 4fb87bcbabecb7cb74a685c512d950c95b88aad8)
```

**Pull request merged**